components/teleport/Teleport.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <section>
+    <Loader v-model="isLoading" />
+    <b-field class="" label-position="inside" label="Select origin parachain">
+      <b-select
+        v-model="fromChain"
+        expanded
+        placeholder="Select parachain 1"
+        required>
+        <option v-for="chain in chains" :key="chain">{{ chain }}</option>
+      </b-select>
+    </b-field>
+
+    <b-field
+      class=""
+      label-position="inside"
+      label="Select destination parachain">
+      <b-select
+        v-model="toChain"
+        disabled
+        expanded
+        placeholder="Select parachain 2"
+        required>
+        <option v-for="chain in chains" :key="chain">{{ chain }}</option>
+      </b-select>
+    </b-field>
+
+    <b-field
+      class=""
+      label-position="inside"
+      :label="`Input recipient address (${toChain})`">
+      <b-input v-model="toAddress" expanded required></b-input>
+    </b-field>
+
+    <b-field class="" label-position="inside" label="Select currency">
+      <b-select
+        v-model="currency"
+        expanded
+        placeholder="Select currency"
+        required>
+        <option v-for="currency in currencies" :key="currency">
+          {{ currency }}
+        </option>
+      </b-select>
+    </b-field>
+
+    <b-field class="" label-position="inside" label="Input currency amount">
+      <b-input
+        v-model="amount"
+        expanded
+        type="number"
+        :min="0.0001"
+        :step="0.0001"></b-input>
+    </b-field>
+
+    <b-button
+      class="button"
+      expanded
+      type="is-primary"
+      label="Send transaction"
+      pack="fas"
+      icon-right="file-import"
+      @click="sendXCM(accountId)" />
+  </section>
+</template>
+
+<script setup lang="ts">
+import { ApiPromise, WsProvider } from '@polkadot/api'
+import { web3Enable } from '@polkadot/extension-dapp'
+import '@polkadot/api-augment'
+import { toDefaultAddress } from '@/utils/account'
+import { getAddress } from '@/utils/extension'
+import { Chain, ChainIdMap } from '@/utils/teleport'
+import { notificationTypes, showNotification } from '@/utils/notification'
+import useAuth from '@/composables/useAuth'
+import Loader from '@/components/shared/Loader.vue'
+import * as paraspell from '@paraspell/sdk'
```

Comment:
Plese use `sub-api` :)

Reply:
I'm afraid that it could not use `sub-api`. Because If I want to teleport tokens from Kusama, I need to use `ApiPromise` of kusama. On the other hand, I would use `ApiPromise` of basilisk. 

And `sub-api` would only return the API endpoint according to the current prefix.  👀

Reply:
https://github.com/kodadot/packages/issues/78